docs/framework/wcf/diagnostics/performance-counters/index.md

```diff
@@ -114,15 +114,11 @@ config.Save();
 ## Programming the WCF Performance Counters  
  Several files are installed in the SDK install folder so that you can access the WCF performance counters programmatically. These files are listed as follows.  
```

Comment:
```suggestion
 Several files are installed in the SDK install folder so that you can access the WCF performance counters programmatically. These files are listed as follows:
```